ipt_connect/IPTdev/parameters.py

```diff
@@ -98,15 +89,13 @@
 # Do we display coreporters publicly?
 display_coreporters = True
 
-
```

Comment:
The blank lines should not have been removed :(
Was this done by your IDE automatically?